README.md

```diff
@@ -1,2 +1,107 @@
-# eslint-udemy
-ESLint configs and plugins used by Udemy.
+Udemy ESLint Packages
+=====================
+
+This repository has [*ESLint*](https://www.github.com/eslint/eslint) related packages used by various Udemy projects. 
+It is owned by [@udemy/team-f](https://github.com/orgs/udemy/teams/team-f), but everyone is welcome to contribute.
+Please refer to [Contributing](#contributing) section to learn more.
+
+This repository uses [*Lerna*](https://github.com/lerna/lerna) to manage multiple [*npm*](https://www.npmjs.com/) packages. We 
+configured Lerna to use [*Yarn Workspaces*](https://yarnpkg.com/lang/en/docs/workspaces/) under the hood.
+
+# Installation
```

Comment:
If you change the title to use #, then change the other things to use ## ?

Reply:
Same comment for the other files.

Reply:
Will keep =====